packages/helldoc-core/src/app/components/Toc.tsx

```diff
@@ -23,12 +23,12 @@ class HeadingLink extends React.PureComponent<HeadingLinkProps> {
     );
   }
 
-  private getLinkProps({ text }: HeadingData) {
+  private getLinkProps({ text, id }: HeadingData) {
     const clearText = text.replace(/\(.*\)/, "");
```

Comment:
What does `clearText` mean , can add some comment here ?  Look like want to remove `\` ?